fluid/resnet50.py

```diff
@@ -170,12 +170,16 @@ def run_benchmark(model, args):
 
     iter = 0
     im_num = 0
+    start_time = time.time()
     for pass_id in range(args.pass_num):
         accuracy.reset(exe)
         if iter == args.iterations:
-            break
+            iter = 0
         for batch_id, data in enumerate(train_reader()):
-            if iter == args.skip_batch_num:
+            if iter < args.skip_batch_num:
+                iter += 1
+                continue
+            if pass_id == 0:
                 start_time = time.time()
```

Comment:
For all batches in pass 0, this line will be executed. Does this is what you want?

Reply:
modified